### PR TITLE
Fix texts in StringsJapanese.kt

### PR DESCRIPTION
**Pull request by salmon-21**
I have fixed typos and unnatural parts and unified the expressions.(I use deepl translation)

### CODE REVIEW

File: src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt [hunk 1]
```diff
@@ -462,9 +464,9 @@ fun Strings.ja(): String? = when (this) {
     FilterStarred -> "スターマークのあるエントリのみ表示"
     FilterUnstarred -> "スターマークのないエントリのみ表示"
     FilterStarIgnored -> "スターマークを参照しない"
-    FilterDone -> "完成のエントリのみ表示"
+    FilterDone -> "完成したエントリのみ表示"
     FilterUndone -> "未完成のエントリのみ表示"
-    FilterDoneIgnored -> "完成状態を参照しない"
+    FilterDoneIgnored -> "完成状態でフィルターしない"
```
Comment (sdercolin) [inline src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt:469]: `FilterStarIgnored` and `FilterDoneIgnored` should use similar phrases for consistency

File: src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt [hunk 2]
```diff
@@ -330,19 +332,19 @@ fun Strings.ja(): String? = when (this) {
...
-    PreferencesEditorPlayerLockedDragUseLabeler -> "ラベラーに従う"
+    PreferencesEditorPlayerLockedDragUseLabeler -> "ラベラーの設定"
     PreferencesEditorPlayerLockedDragUseStart -> "開始位置をドラッグするとき"
     PreferencesEditorPlayerLockedDragNever -> "有効にしない"
     PreferencesEditorPlayerLockedSettingParameterWithCursor -> "カーソル位置に設定する場合も連動ドラッグを有効にする"
     PreferencesEditorPlayerLockedSettingParameterWithCursorDescription ->
-        "「パラメータをカーソル位置に設定」キーボード操作でパラメータを設定するときにも、上記の連動ドラッグ設定を適用します。"
-    PreferencesEditorNotes -> "備考"
+        "キー操作の「パラメータをカーソル位置に設定」でパラメータを設定するときにも、上の連動ドラッグ設定を適用します。"
+    PreferencesEditorNotes -> "メモ"
```
Comment (sdercolin) [inline src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt:347]: Please keep this item unchanged. `備考` is used in other documentations and texts

File: src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt [hunk 3]
```diff
@@ -294,12 +296,12 @@ fun Strings.ja(): String? = when (this) {
     PreferencesChartsSpectrogramWindowType -> "ウィンドウ関数"
     PreferencesChartsSpectrogramColorPalette -> "配色"
     PreferencesKeymap -> "キーマップ"
-    PreferencesKeymapDescription -> "キーボード/マウス操作のキーマップを編集します。"
+    PreferencesKeymapDescription -> "キーボード/マウス操作のキーマップをカスタマイズします。"
```
Comment (sdercolin) [inline src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt:299]: For consistency, please edit all the other `編集します` instances, or keep them unchanged.

**Review by sdercolin — CHANGES_REQUESTED** [review]
Thank you! basically LGTM, please check comments.

File: src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt (hunk 2 above)
Comment (salmon-21) [reply src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt]: updated.

File: src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt (hunk 3 above)
Comment (salmon-21) [reply src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/string/StringsJapanese.kt:299]: updated.

**Review by sdercolin — APPROVED** [review]
LGTM!

**Pull request merged**